### ISSUE TITLE
Add "Furkot Trips" API

### ISSUE THREAD

**Issue by pirxpilot** [APIs-guru/openapi-directory#109]
**Format**: OpenAPI(fka Swagger) 2.0
**Official**: YES
**Url**: https://help.furkot.com/widgets/furkot-api.yaml
**Name**: Furkot Trips

**Comment by IvanGoncharov** [comment]
@pirxpilot Thank you for contribution.
Can you please provide stable URL to service logo?
Any image format will work :)

**Comment by pirxpilot** [comment]
Any those OK? Thanks for looking into that.

![Furkot banner](https://cdn.furkot.com/img/furkot-banner-4x1.svg)
![Furkot banner black](https://cdn.furkot.com/img/furkot-banner-black-4x1.svg)

**Comment by IvanGoncharov** [comment]
@pirxpilot Both looks great 👍
IMHO, black and white is more universal.
Can you please host one of them somewhere and provide stable URL.
In this case, if you change your logo it will be automatically updated in this collection.

**Comment by pirxpilot** [comment]
We *are* hosting them. Github's URL wizardy does a good job hiding that fact though :-)
Let me try again:

https://cdn.furkot.com/img/furkot-banner-4x1.svg
https://cdn.furkot.com/img/furkot-banner-black-4x1.svg
